detail-editace.php

```diff
@@ -0,0 +1,15 @@
+<?php
+require_once "hlavicka.php";
+
+if (!$uzivatelPrihlaseny) {
+    require_once "prihlaseni-formular.php";
```

Comment:
Hele to control flow je imho moc složité. Proč neudělat jen redirect? A nechat formulář na jednom místě, nepřesouvat do samostatného souboru, a ze všech jiných míst ridirectovat na to prihlas-me, kdyz nebudeš mít session? Jako ok, tak jim vysvětleme ten redirect (stejně tam bude), ale pak si to teda na víc místech zkusme a použijme.